packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -0,0 +1,494 @@
+import { Promise } from 'es6-promise';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as ReactTestUtils from 'react-addons-test-utils';
+import { SpinButton } from './SpinButton';
+import { KeyCodes } from '../../Utilities';
+
+const expect: Chai.ExpectStatic = chai.expect;
+
+describe('SpinButton', () => {
+  function renderIntoDocument(element: React.ReactElement<any>): HTMLElement {
+    const component = ReactTestUtils.renderIntoDocument(element);
```

Comment:
yeah, I understand that, perhaps we should just do a separate work to migrate all UTs to enzyme.